src/components/WithForwardRef/withForwardRef.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'React';
+
+export interface WithForwardRefProps<T extends HTMLElement = HTMLElement> {
+  innerRef?: React.Ref<T>;
+}
+
+export function withForwardRef<T extends HTMLElement, P = {}>(
+  Component: React.ComponentType<P>,
```

Comment:
I don't like to use HOC's but I guess there's not a better way to do this.
The main two problems with HOC's are:

1. debugging - We should probably add a different display name (https://reactjs.org/docs/forwarding-refs.html#displaying-a-custom-name-in-devtools).
2. public/static methods and fields won't be externalized

So we need to see this works correctly (maybe add tests for this)

Reply:
@mbutsykin 
I'm not sure why, but it seems the build is breaking because the TextArea component's displayName isn't being copied correctly.
Can you try to see what's the problem here?
you can run `npm run autotools:sanity` to see the failing test